cmd/nirvana/project/init.go

```diff
@@ -638,6 +664,33 @@ required = ["github.com/caicloud/nirvana/utils/api"]
 `
 }
 
+func (o *initOptions) templateGomod() string {
+	return `
+{{- if .Boilerplate -}}
+{{ .SlashBoilerplate }}
+//
+{{ end -}}
+// go.mod example
+//
+// Refer to https://github.com/golang/go/wiki/Modules#gomod
+// for detailed go.mod and go mod command documentation.
+//
+// module github.com/my/module/v3
+//
+// require (
+//     github.com/some/dependency v1.2.3
+//     github.com/another/dependency v0.1.0
+//     github.com/additional/dependency/v4 v4.0.0
+// )
+
+module {{ .ProjectName }}
```

Comment:
`ProjectName` is only the name of your project. But in the example, `module` is path to your project. Should this field be `ProjectPackage`?

Reply:
Yes, you're right.